packages/Select/src/dropdown.scss

```diff
@@ -0,0 +1,50 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+
+:root {
+  /* Default */
+  --ids-dropdown-background: #{tokens.$samoyed};
+  --ids-dropdown-border-radius: #{tokens.$border-radius-sm};
+  --ids-dropdown-zindex: #{tokens.$z-index-100};
+  --ids-dropdown-min-height: #{tokens.$space-1};
+  --ids-dropdown-box-shadow: #{tokens.$shadow-12};
+
+  /* Click Handler */
+  --ids-dropdown-zindex-click-handler: #{tokens.$z-index-90};
+}
+
+.ids-dropdown {
+  width: 100%;
+  position: absolute;
+  min-height: var(--ids-dropdown-min-height);
+  overflow: hidden;
+  background-color: var(--ids-dropdown-background);
+  border-radius: var(--ids-dropdown-border-radius);
+  box-shadow: var(--ids-dropdown-box-shadow);
+  z-index: var(--ids-dropdown-zindex);
+
+  &--top {
+    transform: translateY(-100%);
+  }
+
+  &--right {
+    // TODO
```

Comment:
Is this todo for another time or something you forgot?

Reply:
It's a TODO for when/if the Dropdown bacame it's own component.
It's maybe overkill for now to have it now because it's trivial that it will be needed base on `Dropdown.tsx`.
![image](https://user-images.githubusercontent.com/4323800/186988526-7bf1b0ed-2f8c-4f17-abf8-61b1f3ff773e.png)

Reply:
oh I get it now :) makes sense